src/components/SearchMap.js

```diff
@@ -34,7 +36,11 @@ const SearchMap = ({mapLoaded, lastKnownRegion, updateRegion, sites}) => {
                     title={title}
                     description={description}
                     coordinate={coordinate}
-                />
+                >
+
+                    <Icon type='material-community' name='tent' size={25} color={campsiteIcon}/>
```

Comment:
Custom Icon inside the `MapView` `Marker`. For now I picked the same icon as our `AddSite` tab icon for consistency and ease.  @amgoodman5 please feel free to change the icon to a different one, or an image instead, or the color. These are all placeholders. Also note that I added the style color `campsiteIcon` so we can change the color in `styles` without having to change the name.